site/content/docs/getting-started/setup-creds.md

```diff
@@ -12,12 +12,12 @@ The following are some examples of the typical permissions that are required.
 | --- | --- |--- |
 |deploy | Deploying to Amazon ECS | AWSCloudFormationFullAccess, AmazonECS_FullAccess, AmazonEC2ContainerRegistryFullAccess, AmazonSSMFullAccess, IAMFullAccess |
 |deploy | Deploying to AWS App Runner| AWSCloudFormationFullAccess, AWSAppRunnerFullAccess, AmazonEC2ContainerRegistryFullAccess, AmazonSSMFullAccess, IAMFullAccess|
-|deploy | Deploying to Elastic Beanstalk (deploy) | AWSCloudFormationFullAccess, AdministratorAccess-AWSElasticBeanstalk, AmazonSSMFullAccess, AmazonS3FullAccess (*required to upload the application bundle*), IAMFullAccess |
-|deploy | Hosting WebAssembly Blazor App in S3 & CloudFront | AmazonS3FullAccess, CloudFrontFullAccess, IAMFullAccess, AmazonSSMFullAccess, AWSLambda_FullAccess (*required to copy from CDKBootstrap bucket to S3 bucket*)|
-| list-deployments | List CF stacks| AWSCloudFormationReadOnlyAccess  |
-| delete-deployment | Delete a CF stack | AWSCloudFormationFullAccess + permissions for resources being deleted |
+|deploy | Deploying to AWS Elastic Beanstalk (deploy) | AWSCloudFormationFullAccess, AdministratorAccess-AWSElasticBeanstalk, AmazonSSMFullAccess, AmazonS3FullAccess (*required to upload the application bundle*), IAMFullAccess |

```

Comment:
It was already here, but not sure why we have the trailing "(deploy") here.

Reply:
Good eyes, I certainly hadn't noticed that detail - fixed with new commit.